aws/sagemaker/studio/gpu.Dockerfile

```diff
@@ -0,0 +1,110 @@
+FROM nvidia/cuda:10.1-base-ubuntu18.04
+
+ARG NB_USER="sagemaker-user"
+ARG NB_UID="1000"
+ARG NB_GID="100"
+
+RUN apt-get update || true && \
+    apt-get install -y sudo && \
+    useradd -m -s /bin/bash -N -u $NB_UID $NB_USER && \
+    chmod g+w /etc/passwd && \
+    echo "${NB_USER}    ALL=(ALL)    NOPASSWD:    ALL" >> /etc/sudoers && \
+    # Prevent apt-get cache from being persisted to this layer.
+    rm -rf /var/lib/apt/lists/*
+
+ARG DEBIAN_FRONTEND=noninteractive
+
+RUN apt-get update && \
+    apt-get install -y locales && \
+    sed -i -e 's/# en_US.UTF-8 UTF-8/en_US.UTF-8 UTF-8/' /etc/locale.gen && \
+    dpkg-reconfigure --frontend=noninteractive locales && \
+    update-locale LANG=en_US.UTF-8
+
+RUN apt-get install -y \
+    openjdk-11-jdk-headless \
+    python3-pip git curl unzip
+
+RUN ln -s /usr/bin/python3 /usr/bin/python & \
+    ln -s /usr/bin/pip3 /usr/bin/pip
+
+RUN pip install jupyter ipykernel awscli
+
+#### ADDing GPU Stuff, REMOVE unnecessary stuff later
+ENV PYTHONDONTWRITEBYTECODE=1 \
+    PYTHONUNBUFFERED=1 \
+    LD_LIBRARY_PATH="${LD_LIBRARY_PATH}:/usr/local/lib" \
+    PYTHONIOENCODING=UTF-8 \
+    LANG=C.UTF-8 \
+    LC_ALL=C.UTF-8 \
+    SAGEMAKER_TRAINING_MODULE=sagemaker_mxnet_container.training:main \
+    DGLBACKEND=mxnet \
+    CUDNN_VERSION=7.6.0.64-1+cuda10.1 \
+    NCCL_VERSION=2.4.8-1+cuda10.1
+
+RUN apt-get update \
```

Comment:
Why we install additional things here? The CUDA base image should be sufficient enough right? Please check the code build setup we had.